index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Projeto</title>
```

Comment:
Filipe, o que você acha de alterar o Título utilizando o título completo do projeto. Isso ajuda a identificar melhor de qual projeto esse arquivo pertence.
```suggestion
  <title>Projeto Lessons Learned</title>
```